exercism/kindergarten-garden/kindergarten-garden.js

```diff
@@ -0,0 +1,50 @@
+var Garden = function(diagram, students=[
+  'Alice','Bob','Charlie','David',
+  'Eve','Fred', 'Ginny', 'Harriet',
+  'Ileana', 'Joseph', 'Kincaid', 'Larry'
+]) {
+  this.diagram = diagram.split('\n')
+  this.students = students.sort()
```

Comment:
This must be because I'm new or something, but what kind of sort() is happening here? I feel a comment might be helpful :)